### ISSUE TITLE
Error analyzing mp3 file with chapters: "Caught exception: Undefined array key 1"

### ISSUE THREAD

**Issue by yassinedoghri** [JamesHeinrich/getID3#355]
When analyzing an mp3 file with chapters, I get the following error: 
The returned data from the analyze function is missing a chunk of the info, such as `mime_type`, `avdataoffset`, ...  

### Steps to reproduce

1. Download the following mp3: https://www.buzzsprout.com/1529720/6705454-cc-1-introductions.mp3
2. Analyze it using the $getID3->analyze($file) method.
3. Dump result and see error with missing data.

### Expected behavior

The returned data from the analyze method should include all of the expected data.

### Relevant logs and/or screenshots

What is returned:
![image](https://user-images.githubusercontent.com/11021441/146210480-5579631a-c105-43a9-903d-9270acbd7b96.png)

What should be returned:
![image](https://user-images.githubusercontent.com/11021441/146210657-e43c8f99-5cd3-4863-8cce-8878962422eb.png)

### Context

- getID3: v2.0.0-beta4
- PHP v8.0.13

### Possible fixes

With some _brute force var_dumps_, I was able to pinpoint the issue to this line, just after the `WXXX` case:
```php
// v1.9 - getid3/module.tag.id3v2.php - line 2067
// v2 - src/Module/Tag/ID3v2.php - line 2066
list($subframe['chapter_url_description'], $subframe['chapter_url']) = explode("\x00", $encoding_converted_text, 2);
```
👉 Apparently, the `$encoding_converted_text` doesn't include the `chapter_url` so the `index 1` doesn't exist in the array created from the `explode()`.

**Naively**, this fixes it:
```php
$encoding_converted_text_array = explode("\x00", $encoding_converted_text, 2);
$subframe['chapter_url_description'] = $encoding_converted_text_array[0];
$subframe['chapter_url'] = isset($encoding_converted_text_array[1]) ? $encoding_converted_text_array[1] : '';
```
---

That being said, **thanks** for your hard work, getID3 is amazing  🙂

**Comment by JamesHeinrich** [comment]
You correctly traced the source of the problem. Even more naively, error-suppressing the `list` will fix it:
`@list($subframe['chapter_url_description'], $subframe['chapter_url']) = explode("\x00", $encoding_converted_text, 2);`
Which is how I have fixed it in https://github.com/JamesHeinrich/getID3/commit/a99e0a20540259911e95e3bcfdd18db2ba8222dc

**Comment by StudioMaX** [comment]
I backported this fix to 2.0 in #356.
Please require `james-heinrich/getid3:^2.0-dev` if you need so in your composer.json until the new version is released.
